clusters/veritable-prod/alice/cloudagent/release.yaml

```diff
@@ -0,0 +1,29 @@
+---
+apiVersion: helm.toolkit.fluxcd.io/v2beta1
+kind: HelmRelease
+metadata:
+  name: cloudagent
+  namespace: alice
+spec:
+  install:
+    remediation:
+      retries: -1
+  releaseName: cloudagent
+  chart:
+    spec:
+      version: 1.0.6
+      chart: veritable-cloudagent
+      sourceRef:
+        kind: HelmRepository
+        name: digicatapult
+  interval: 10m0s
```

Comment:
what does this do?

Reply:
This installs the veritable cloudagent in the `alice` namespace

Reply:
To set how long between attempts to reconcile the helmRelease.